ci: Fix several vulnerabilities from dependabot

hlomzik opened the pull request:
Fixes #4498 #4536
- https://github.com/HumanSignal/label-studio-frontend/pull/1549
- https://github.com/HumanSignal/dm2/pull/229

dependabot[bot] committed:
chore(deps): bump semver from 5.7.1 to 5.7.2 in /label_studio/frontend

Bumps [semver](https://github.com/npm/node-semver) from 5.7.1 to 5.7.2.
- [Release notes](https://github.com/npm/node-semver/releases)
- [Changelog](https://github.com/npm/node-semver/blob/v5.7.2/CHANGELOG.md)
- [Commits](https://github.com/npm/node-semver/compare/v5.7.1...v5.7.2)

---
updated-dependencies:
- dependency-name: semver
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
chore(deps): bump word-wrap in /label_studio/frontend

Bumps [word-wrap](https://github.com/jonschlinkert/word-wrap) from 1.2.3 to 1.2.4.
- [Release notes](https://github.com/jonschlinkert/word-wrap/releases)
- [Commits](https://github.com/jonschlinkert/word-wrap/compare/1.2.3...1.2.4)

---
updated-dependencies:
- dependency-name: word-wrap
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

hlomzik committed:
ci: Fix hexo-utils back to fix dependabot issue

hlomzik committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/6006494373

hlomzik committed:
[submodules] Build static HumanSignal/dm2

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/6006554741

robot-ci-heartex commented:
To enable Auto Merge for this PR also merge those PRs:
- https://github.com/HumanSignal/label-studio-frontend/pull/1549

hlomzik committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/6010901220

hlomzik committed:
Merge branch 'origin/develop' into 'fb-lsdv-5474/dependabot'
